src/components/meta/VButton.stories.mdx

```diff
@@ -0,0 +1,120 @@
+import {
+  ArgsTable,
+  Canvas,
+  Description,
+  Meta,
+  Story,
+} from '@storybook/addon-docs'
+
+import VButton, {
+  buttonForms,
+  buttonSizes,
+  buttonVariants,
+} from '~/components/VButton.vue'
+
+<Meta
+  title="Components/VButton"
+  components={VButton}
+  argTypes={{
+    as: {
+      options: buttonForms,
+      control: { type: 'radio' },
+    },
+    variant: {
+      options: buttonVariants,
+      control: { type: 'select' },
+    },
+    pressed: { control: 'boolean' },
+    size: {
+      options: buttonSizes,
+      control: { type: 'radio' },
+    },
+    disabled: { control: 'boolean' },
+    focusableWhenDisabled: { control: 'boolean' },
+    type: { control: 'text' },
+  }}
+/>
+
+export const Template = (args) => ({
+  template: `
+    <VButton v-bind="args" @click="onClick" href="/">
+      Code is Poetry
+    </VButton>
+  `,
+  components: { VButton },
+  methods: {
+    onClick() {
+      window.alert('hello!')
+    },
+  },
+  setup() {
+    return { args }
+  },
+})
+
+# VButton
+
+<Description of={VButton} />
+
+<ArgsTable of={VButton} />
+
+<Canvas>
+  <Story name="VButton" args={{}}>
+    {Template.bind({})}
+  </Story>
+</Canvas>
+
+## Button variants
+
+export const VariantsTemplate = (args) => ({
+  template: `<div class="flex gap-4 flex-wrap">
+    <VButton v-bind="args" variant="primary">
```

Comment:
It would be nice to `v-for` this so it automatically updates for future variants (or if we remove any).

Reply:
Thank you for nudging me to actually do this :)